class LoadingFailedException(Exception):
    pass


class SavingFailedException(Exception):
    pass
